docs/guides/configuration-guide/loadbalancer.md

```diff
@@ -25,42 +29,76 @@ hosts_additional_entries:
 
 ## TLS certificates
 
-To enable external TLS encryption:
+:::warning
+To avoid unnecessary additional work and problems, it is recommended that you configure TLS with the intended target
+configuration of the specific environment during the initial rollout process.
+:::
 
-```yaml title="environments/kolla/configuration.yml"
-kolla_enable_tls_external: "yes"
-```
+For example: Changes to the configuration of TLS or the fully qualified domain names (FQDNs) will result in new URLs (with and without the https prefix)
+stored in the Openstack databases, and in the case of self-signed certificates, the CA certificate must be distributed to all participating clients.
```

Comment:
This doesn't explain anything really.

When using self-signed certificates, you always need to distribute/update the CA if anything changes?

Which specific changes to the TLS configuration results in new URLs? Even as a maintainer of this software I would have to look this up, so it should be explained to the operator configuring this, because they certainly won't know.

Please add something why this is a specific problem, and if it is a problem, how it can be fixed (e.g. new URLs in the DB), or if it can't be fixed easily, just say so.

If we warn about something the warning must be understandable without some secret knowledge, and it must be actionable: What should the operator do, or what should they avoid doing?

Thanks!